charts/loki-distributed/README.md

```diff
@@ -38,6 +38,10 @@ helm repo add grafana https://grafana.github.io/helm-charts
 | compactor.podAnnotations | object | `{}` | Annotations for compactor pods |
 | compactor.priorityClassName | string | `nil` | The name of the PriorityClass for compactor pods |
 | compactor.resources | object | `{}` | Resource requests and limits for the compactor |
+| compactor.serviceAccount.annotations | object | `{}` | Annotations for the compactor service account |
+| compactor.serviceAccount.create | bool | `false` |  |
+| compactor.serviceAccount.imagePullSecrets | list | `[]` | Image pull secrets for the compactor service account |
+| compactor.serviceAccount.name | string | `nil` | The name of the ServiceAccount to use for the compactor. If not set and create is true, a name is generated using the fullname template |
```

Comment:
That's not quite correct. It falls back to using the common service account as configured.

Reply:
*if* create is set, it will create one named $serviceAccountName-compactor, so it is correct. If you set create=false (the default), it will fall back to using the common service account.

Reply:
The fallback is not the fullname template. It's the template for the service account name which in turn defaults to the fullname template.

Reply:
I misunderstood you to mean that it used the common serviceaccount, not the common serviceaccount name. I've updated the docs.